tests/unit/accordion/methods.js

```diff
@@ -10,6 +10,18 @@ var equalHeight = testHelper.equalHeight,
 
 module( "accordion: methods", setupTeardown() );
 
+test( "collapsedHeight", function() {
```

Comment:
You're testing a set of options, not a method, so this is in the wrong file and the test title doesn't really make sense.